tools/packaging/kata-deploy/local-build/kata-deploy-build-and-upload-payload.sh

```diff
@@ -5,7 +5,7 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-KATA_DEPLOY_DIR="`dirname ${0}`/../../kata-deploy-cc"
```

Comment:
Curious why the target issue #6343 is already shown as closed? Is it the correct one?

Reply:
Out of lazyness, to be honest, as all the fixes are actually related to that issue.

Reply:
OK.